features_and_tools/restful_api.rst

```diff
@@ -23,6 +23,10 @@ Firely Server enables create-on-update: If you request an update and no resource
 
 Firely Server can reject a resource based on :ref:`feature_prevalidation`.
 
+Update with no changes
+Updating a resource with the same content that is already present on a server will produce a "No-Op" outcome. The server will respond successfully, but no changes will actually be stored.
+If you want to know whether your operation was a No-Op or not, you can examine the OperationOutcome resource that is returned by the server. You may need to set the Prefer Header in order to instruct the server to return an outcome.
```

Comment:
Maybe good to give an example here on how you would set the Prefer Header to achieve this.

Reply:
added